go-apps/meep-metrics-engine/server/api_metrics.go

```diff
@@ -31,3 +31,11 @@ import (
 func MetricsGet(w http.ResponseWriter, r *http.Request) {
```

Comment:
move generated code to version-specific folders

Reply:
fixed